README.md

```diff
@@ -29,7 +29,7 @@ kubecolor internally calls `kubectl` command and try to colorizes the output so
 you can use kubecolor as a complete alternative of kubectl. It means you can write this in your .bash_profile:
 
 ```sh
-alias kubectl="kubecolor"
+command -v kubecolor >/dev/null 2>&1 && alias kubectl="kubecolor"
```

Comment:
Yeah, this does not break the alias. However, I don't think this is not needed because of 2 reasons:

* Users cannot be aware missing kubecolor on the $PATH.
* kubecolor will be likely on the $PATH, because they want to use kubecolor. In other words, users should be responsible to install the command on the $PATH before using it.

How about just writing "If the alias doesn't work, please make sure `kubecolor` is installed on your $PATH."?

Reply:
I should have explained my use case: If you use your `.bash_profile` or `.zshrc` or whatever on more than one computer you might not have `kubecolor` installed everywhere.
For this case it turns out to be helpful to check if a command is on the `PATH` before setting the alias, which would make `kubectl` useless.

Reply:
I see, it makes sense. However, I would say some users might get confused because they are not sure why it's checking kubecolor existence while just `alias kubectl...` simply looks ok. So I'd prefer adding another line below current one to describe it as you suggested. Can you do that.

Reply:
:+1: 
How about  eb02704 ?

Reply:
Cool